### PR TITLE
Fix docs CI

### LINKED ISSUE #222: Docs CI fails
need to check and fix

### PR DESCRIPTION
**Pull request by mmaelicke**
The PR should fix the still failing documentation CI

closes #222

### CODE REVIEW

**Comment by mmaelicke** [comment]
@AlexDo1, now all tests pass and docs are build correctly. You can review, please

File: .github/workflows/pages.yml [hunk 1]
```diff
@@ -33,14 +33,15 @@ jobs:
       
       - name: Build Sphinx
         run: |
-          pip install sphinx pandoc pydata_sphinx_theme nbsphinx sphinx-gallery
+          pip install "sphinx<6" pandoc pydata_sphinx_theme nbsphinx sphinx-gallery
           cd docs
           make html
+          make html
```
Comment (AlexDo1) [inline .github/workflows/pages.yml:39]: This was the 'ugly hotfix' right? Do we want to keep that?

File: .github/workflows/pages.yml [hunk 2]
```diff
@@ -33,14 +33,15 @@ jobs:
...
+          pip install "sphinx<6" pandoc pydata_sphinx_theme nbsphinx sphinx-gallery
           cd docs
           make html
+          make html
           cd build/html
           touch .nojekyll
       
       -  name: Deploy
-         if: success()
+         if: success() || true
```
Comment (AlexDo1) [inline .github/workflows/pages.yml:44]: This was also part of the 'ugly hotfix', keep it?

**Review by AlexDo1 — COMMENTED** [review]
So many new blank lines, nice!

Looks good, I only had two comments about the CI.

File: .github/workflows/pages.yml (hunk 1 above)
Comment (mmaelicke) [reply .github/workflows/pages.yml:39]: yep...

**Review by AlexDo1 — APPROVED** [review]
Documentation workflow is still successful, nice!

**Pull request merged**